tgstation.dme

```diff
@@ -2757,7 +2757,7 @@
 #include "code\modules\research\designs\autolathe_desings\autolathe_designs_tcomms_and_misc.dm"
 #include "code\modules\research\designs\autolathe_desings\autolathe_designs_tools.dm"
 #include "code\modules\research\designs\comp_board_designs\comp_board_designs_all_misc.dm"
-#include "code\modules\research\designs\comp_board_designs\comp_board_designs_cargo .dm"
+#include "code\modules\research\designs\comp_board_designs\comp_board_designs_cargo.dm"

```

Comment:
The reason for this change here is that the space at the end of the file actually caused issues with the conversion, because space before file extensions fucks things over. If you are web-editing files together, **PLEASE** make sure you name them properly.

Reply:
I might know whom to blame for this change...